internal/cloudapi/v2/compose.go

```diff
@@ -898,6 +899,7 @@ func (request *ComposeRequest) GetBlueprintFromCustomizations() (blueprint.Bluep
 		bp.Customizations = nil
 	}
 
+	bp.CryptPasswords()
```

Comment:
We will need to check for error, maybe even wrap the returned error, my suggestion is without wrapping :)
```go
err = bp.CryptPasswords()
if err != nil {
  return bp, err
}
```